res/controllers/Reloop-Jockey-3-ME-scripts.js

```diff
@@ -452,9 +446,9 @@ Jockey3ME.MixerVol = function (channel, control, value, status, group) {
     currentDeck += 2;
   }
   if (control == 0x2D || control == 0x6C) {
-	  engine.setValue("[Channel" + currentDeck + "]","pregain",script.absoluteNonLin(value,0,1,4,0,127));
+	  engine.setValue("[Channel" + currentDeck + "]","pregain",script.absoluteLin(value,0,script.absoluteLin(value,0,4,0,127),0,127));
```

Comment:
Until https://bugs.launchpad.net/mixxx/+bug/1536796 is fixed, you can put a line with engine.setParameter() next to this but commented out.